File: miguel/(bernardo)/Problema0256/problema0256.py
# Entrada de valores e ordenação
vetor = []
for x in range(8):
    numero = int(input(f"Digite o {x + 1} número: "))
    for n in range(x):
        if numero < vetor[n]:
            intermediario = vetor[n]
            vetor[n] = numero
            numero = intermediario
    vetor.append(numero)

# Saída
print(f"O vetor ordenado é: {vetor}")

File: miguel/(bernardo)/Problema0256/problema0256.js
const prompt = require("prompt-sync")({ sigint: true });

// Entrada de valores e ordenação
var vetor = [];
for (var  i = 0; i < 8; i++){
    var numero = parseInt(prompt("Digite o "+ (i+1) +" número: "));
    for (var  j = 0; j < i; j++){
        if (numero < vetor[j]){
           var intermediario = vetor[j];
           vetor[j] = numero;
           numero = intermediario;
        }
    }
    vetor[i] = numero;
}

// Saída
console.log("O vetor ordenado é: " + vetor);